src/app/pages/details/details.js

```diff
@@ -0,0 +1,76 @@
+        let title = decodeURIComponent(window.location.search.substr(1));
+
+        this.getThisTitle = new GetThisTitle(title);
+        this.model.set('bookInfo', {
+            title,
+            blurb: `College Physics meets standard scope and sequence requirements for a
+                two-semester introductory alegabra-based physics course. The text is
+                grounded aliqua salami tongue fugiat anim. Andouille sunt labore sint,
+                incididunt turkey tri-tip swine pork chop jowl cow fatback spare ribs shank
+                enim. Commodo cow laborum ad adipisicing spare ribs excepteur.`,
+            mainAuthorTitle: 'Senior Contributing Authors',
+            mainAuthorHtml: `
+                 <p>Dr. Paul Peter Urone, California State University Sacramento</p>
+                 <p>Dr. Roger Hinrichs, State University of New York, College at Oswego</p>
+            `,
+            allAuthorsHtml: `
+                 <h2>Senior Contributing Authors</h2>
+                 <p>Dr. Paul Peter Urone, California State University Sacramento</p>
+                 <p>Dr. Roger Hinrichs, State University of New York, College at Oswego</p>
+                 <h2>Contributing Authors</h2>
+                 <p>Dr. Paul Peter Urone, California State University Sacramento</p>
+                 <p>Dr. Roger Hinrichs, State University of New York, College at Oswego</p>
+            `,
+            endorsement: `We are adopting the Physics text and I love it. We use mostly the
+                Kinematics chapters, but wow, what a great way to get this information to
+                people in our field.`,
+            attribution: 'Maureen Caroll, ASET',
+            facultyResources: [
+                {
+                    title: 'Getting Started Guide for Faculty',
+                    iconUrl: 'http://wwwimages.adobe.com/content/dam/acom/en/legal/images/badges/Adobe_PDF_file_icon_32x32.png',
+                    downloadUrl: '/downloads'
+                },
+                {
+                    title: 'PowerPoint Slides',
+                    iconUrl: 'http://icons.iconarchive.com/icons/dakirby309/windows-8-metro/256/Office-Apps-PowerPoint-alt-1-Metro-icon.png',
+                    downloadUrl: '/downloads'
+                },
+                {
+                    title: 'Sample Syllabus Language',
+                    iconUrl: 'http://wwwimages.adobe.com/content/dam/acom/en/legal/images/badges/Adobe_PDF_file_icon_32x32.png',
+                    downloadUrl: '/downloads'
+                },
+                {
+                    title: 'Instructor Solution Manual',
+                    iconUrl: 'http://wwwimages.adobe.com/content/dam/acom/en/legal/images/badges/Adobe_PDF_file_icon_32x32.png',
+                    downloadUrl: '/downloads'
+                }
+            ]
+        });
+    }
+
+    onRender() {
+        this.el.classList.add('text-content');
+        this.regions.getThisTitle.show(this.getThisTitle);
```

Comment:
If this view is being completely redrawn, any views within it probably should as well.  Is creating the `getThisTitle` view in this view's constructor an attempt to save state, or is it just to avoid re-creating it?  Note: it will still be re-rendered from `.show()`.